.github/workflows/test-dbt-installation-docker.yml

```diff
@@ -103,66 +89,43 @@ jobs:
+        id: installation-test
         uses: addnab/docker-run-action@v3
         with:
           registry: ${{ env.GITHUB_PACKAGES_LINK }}
           image: ${{ steps.image-info.outputs.ref }}
           run: |
-            PACKAGE=$(echo ${{ inputs.package_name }} | cut -c 5-)
-            python -c "import dbt.adapters.$PACKAGE"
-
-      - name: "Dump Job Status"
-        if: ${{ always() }}
-        id: job-status
+            if [[ ${{ inputs.package_name }} == "dbt-core" ]]
+            then
+              dbt --version
+            else
+              PACKAGE=$(echo ${{ inputs.package_name }} | cut -c 5-)
+              python -c "import dbt.adapters.$PACKAGE"
+            fi
+
+      - name: "[INFO] Docker installation test failure - GHA"
+        if: ${{ always() && steps.installation-test.outcome != 'success' }}
         run: |
-          file="test-${{ inputs.package_name }}-${{ strategy.job-index }}-${{ matrix.tag }}.json"
-          # Create file
-          touch $file 
-          # Write job status to file
-          echo $JOB_STATUS >> $file
-          # Set path to file for subsequent steps          
-          echo "path=$file" >> $GITHUB_OUTPUT
-        env:
-          JOB_STATUS: ${{ toJson(job.status) }}
+          title="Docker installation test failure"
+          message="Docker nightly installation test failed for ${{ inputs.package_name }}@${{ matrix.tag }}"
+          echo "::notice title=${{ env.NOTIFICATION_PREFIX }}: $title::$message"
 
-      - name: "Upload Job Status For Slack Notification"
-        if: ${{ always() }}
-        uses: actions/upload-artifact@v4
+      - name: "[INFO] Docker installation test failure - Slack"
+        uses: ravsamhq/notify-slack-action@v2
+        if: ${{ always() && steps.installation-test.outcome != 'success' && github.event_name == 'schedule' }}
         with:
-          name: ${{ needs.generate-artifact-name.outputs.artifact-name }}-${{ matrix.tag }}
-          path: ${{ steps.job-status.outputs.path }}
-          retention-days: ${{ env.ARTIFACT_RETENTION_DAYS }}
-
-  send-slack-notification:
-    needs: [generate-artifact-name, docker-installation-test]
-    if: ${{ failure() && github.event_name == 'schedule' }}
-
-    uses: ./.github/workflows/test-dbt-installation-notify-job-statuses.yml
-    with:
-      artifact_name: ${{ needs.generate-artifact-name.outputs.artifact-name }}
-      installation_method: "Docker"
-      package_name: ${{ inputs.package_name }}
-
-    secrets:
-      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+          status: ${{ job.status }}
+          notification_title: "Docker nightly installation test failed for ${{ inputs.package_name }}@${{ matrix.tag }}"
+          message_format: "{emoji} *{workflow}* {status_message} in <{repo_url}|{repo}@{branch}> on <{commit_url}|{commit_sha}>"
+          footer: "<{run_url}|View Run>"
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
This is what the central `dbt-labs/dbt-release/.github/workflows/slack-post-notification.yml@main` does so we don't have to redo this logic everywhere.  Did you not use that for a reason?

Reply:
Oh, I bet you're right that I can put that back in as a reusable workflow. I stripped it out when I was trying to update this to produce a unique artifact path and never revisited consolidating it. I'll update it.